packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -213,6 +213,8 @@ export const useArbTokenBridge = (
           symbol,
           address: l1Address,
           l2Address: address.toLowerCase(),
+          parentBridgeAddress: bridgeInfo[l1NetworkID]?.originBridgeAddress,
```

Comment:
@chrstph-dvx  Are we 100% sure `parent` and `child` here refer to `origin` and `dest` addresses?